### PR TITLE
Populate files array in track config

### PR DESCRIPTION
**Pull request by BNAndras**
The files array should be set per https://exercism.org/docs/building/tracks/config-json#h-files

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -16,9 +16,15 @@
...
-    "test": [],
-    "example": [],
+    "solution": [
+      "%{pascal_slug}.bat"
+    ],
+    "test": [
+      "%{pascal_slug}Test.bat"
+    ],
+    "example": [
+      ".meta/Example.bat"
```
Comment (kotp) [inline config.json:26]: Probably better to not capitalize the name of this file.

```suggestion
      ".meta/example.bat"
```

**Review by kotp — APPROVED** [review]
Requested change, but otherwise good.

**Comment by BNAndras** [comment]
Thanks! I'll leave that one up to the maintainer since that'll affect the existing Example.bat files.

File: config.json (hunk 1 above)
Comment (ErikSchierboom) [reply config.json:26]: It's fine to capitalize as that is how all files are named in this track.

**Review by ErikSchierboom — APPROVED** [review]

**Pull request merged**